src/module.cpp

```diff
@@ -128,3 +130,22 @@ void Module::VerifyFunction() {
 		}
 	}
 }
+
+std::string Module::MapSpecieReact(const std::pair<std::string, int> &specie) {
```

Comment:
Many of these functions don't actually map, but are closer to something like "SpecieToString".